Cargo.toml

```diff
@@ -17,6 +17,6 @@ serde = "1.0"
 serde_derive = "1.0"
 serde_json = "1.0"
 tokio-core = "0.1"
-web3 = { git = "https://github.com/polyswarm/rust-web3.git", branch = "relay" }
+web3 = { git = "https://github.com/polyswarm/rust-web3.git", branch = "fix/non-blocking-poll-confirmations" }
```

Comment:
Is this needed now? Should we update relay branch / push upstream?

Reply:
I changed the branch in our version.

Though, I'm sure the upstream project would appreciate the ability to wait for existing transaction confirmations.

Reply:
I guess it probably isn't necessary. But I like that the wait for confirmations returns an option now. I'm not entirely sure, but I bet that helps us with https://github.com/polyswarm/relay/issues/5

Reply:
It does not help with 5. Looks like the confirm.rs stuff just skips the blocks in the Stream to wait.